.github/actions/generate-conventional-release-notes/closest-version.js

```diff
@@ -2,7 +2,9 @@ const semver = require('semver');
 const fs = require('fs');
 const gitSemverTags = require('git-semver-tags');
 
-const packageJson = JSON.parse(fs.readFileSync('./package.json', 'utf8'));
+const readPackageJson = () => {
+    return JSON.parse(fs.readFileSync('./package.json', 'utf8'));
+}
```

Comment:
Rename this to `getLernaJson` and in package.json remove "version" property and make it `"private": true` since we have monorepo